### PR TITLE
Update macos-notarization-issues.md

### PR DESCRIPTION
**Pull request by agocke**
Note that an apphost is produced by default and ad hoc signed.

### CODE REVIEW

File: docs/core/install/macos-notarization-issues.md
Comment (gewarren) [inline docs/core/install/macos-notarization-issues.md:19]: Can we put a specific version here? E.g. 
```suggestion
In .NET SDK 7 and later versions, an **appHost**, which is a native Mach-O executable, is produced for your app. This executable is usually invoked by .NET when your project compiles, publishes, or is run with the `dotnet run` command. The non-**appHost** version of your app is a _dll_ file that can be invoked by the `dotnet <app.dll>` command.
```

File: docs/core/install/macos-notarization-issues.md
Comment (gewarren) [inline docs/core/install/macos-notarization-issues.md:21]:
```suggestion
When run locally, the SDK signs the apphost using [ad hoc signing](https://developer.apple.com/documentation/security/seccodesignatureflags/1397793-adhoc), which allows the app to run locally. When distributing your app, you'll need to properly sign your app according to Apple guidance.
```

File: docs/core/install/macos-notarization-issues.md
Comment (gewarren) [inline docs/core/install/macos-notarization-issues.md:23]:
```suggestion
You can also distribute your app without the apphost and rely on users to run your app using `dotnet`. To turn off **appHost** generation, add the [`UseAppHost`](../project-sdk/msbuild-props.md#useapphost) boolean setting in the project file and set it to `false`. You can also toggle the appHost with the `-p:UseAppHost` parameter on the command line for the specific `dotnet` command you run:
```

File: docs/core/install/macos-notarization-issues.md
Comment (gewarren) [inline docs/core/install/macos-notarization-issues.md:39]:
```suggestion
An **appHost** is required when you publish your app [self-contained](../deploying/index.md#publish-self-contained) and you cannot disable it.
```

**Review by gewarren — APPROVED** [review]

**Comment by DamianEdwards** [comment]
@agocke @BillWagner is this waiting on something?

**Comment by BillWagner** [comment]
@DamianEdwards @agocke 

It has slipped through the cracks waiting on the review comments. I accepted the suggestions, and this will merge once the build finishes.

**Pull request merged**
